L1Trigger/TrackerTFP/interface/MiniHoughTransform.h

```diff
@@ -0,0 +1,67 @@
+#ifndef L1Trigger_TrackerTFP_MiniHoughTransform_h
+#define L1Trigger_TrackerTFP_MiniHoughTransform_h
+
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+
+#include <vector>
+#include <set>
+#include <deque>
+
+namespace trackerTFP {
+
+  // Class to find in a region finer rough candidates in r-phi
```

Comment:
Make comment describing class more precise. e.g. "Class to refine HT track candidates, by subdividing each HT cell into a finer granularity array"?

Reply:
done